.github/templates/model_regression_test_results.tmpl

```diff
@@ -7,33 +7,25 @@ a markdown table with a summary of the tests.
 */ -}}
 {{- /*
 
-The accuracy_tpl template returns data for the accuracy field.
+The print_result template returns data depends on available fields.
 
 */ -}}
-{{ define "accuracy_tpl" -}}
-{{- if has .master "accuracy" -}}
+{{ define "print_result" -}}
+{{- if and (has (index .branch "micro avg") "f1-score") (has (index .master "micro avg") "f1-score") -}}
+{{ printf "%.4f" (index (index .branch "micro avg") "f1-score") }} ({{ printf "%.2f" ((index (index .master "micro avg") "f1-score") | math.Sub (index (index .branch "micro avg") "f1-score")) }})
+{{- else if and (has .branch "accuracy") (has .master "accuracy") -}}
 {{ printf "%.4f" .branch.accuracy }} ({{ printf "%.2f" (.master.accuracy | math.Sub .branch.accuracy) }})
-{{- else -}}
+{{- else if and (has .branch "accuracy") (has (index .master "micro avg") "f1-score") -}}
+{{ printf "%.4f" .branch.accuracy }} ({{ printf "%.2f" ((index (index .master "micro avg") "f1-score") | math.Sub .branch.accuracy) }})
+{{- else if (has .branch "accuracy") -}}
```

Comment:
Aren't we missing the condition where branch has `micro avg F1` but master has `accuracy`? That shouldn't result in `no data`.

Reply:
Good eye! added

Reply:
@tczekajlo Looking at this again: Am I correct in assuming that the difference calculation will be done as `branch.accuracy - master.accuracy` and not as `master.accuracy - branch.accuracy`?
Since we want to know how far is the performance on branch compared to master it should be the former.

Reply:
Yes, it's `branch.accuracy - master.accuracy`.

```
 $ gomplate -i '{{ ( 3 | math.Sub 5) }}'
2
```